build.sbt

```diff
@@ -243,9 +242,11 @@ lazy val V = new {
   def scala2Versions = nonDeprecatedScala2Versions ++ deprecatedScala2Versions
 
   // Scala 3
-  def scala3RCVersions = Seq("3.0.0-RC3", "3.0.0-RC2", "3.0.0-RC1")
```

Comment:
@smarter is it ok to drop them? I can't find the date that you mentioned previously.

Reply:
Could you wait until next week to drop RC1? We had to move the final exam for our class to tomorrow (Saturday) and are still using 3.0.0-RC1 there.

Reply:
In fact if you can wait until next week for the release in general this ensures that we won't run into any new exciting issue tomorrow :sweat_smile:

Reply:
@smarter Sure. It's a preparation step for release. We aren't going to do release until Monday.

Reply:
can we release on nightly for bleeding edge users 😄

Reply:
@swoorup Unfortunately, that's won't work as we can't publish artifacts per every nightly version.

Reply:
it should be possible to setup an infrastructure for it, but that would require building mtags on the dotty side